.github/workflows/test.yml

```diff
@@ -19,14 +19,16 @@ jobs:
         uses: actions/cache@v3
         with:
           path: ${{ steps.pnpm-cache.outputs.STORE_PATH }}
-          key: ${{ runner.os }}-${{ matrix.node }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          key: ${{ runner.os }}-${{ matrix.node }}-pnpm-store-
           restore-keys: ${{ runner.os }}-${{ matrix.node }}-pnpm-store-k
-      - uses: actions/cache@v3
+      - name: Cache gradle and sourcegraph stores
+        uses: actions/cache@v3
         with:
           path: |
             ~/.gradle/caches
```

Comment:
I wonder - is it ok to cache IntelliJ artefacts? They seem to take a lot of space 

This is an example of a repo that removes them 😄 
https://github.com/VirtusLab/git-machete-intellij-plugin/blob/8910e34c833779b5f8261fca298aeb549b80c875/.circleci/config.yml#L198-208

Reply:
The cache size limit is 10 GB.